contracts/validator_registration.v.py

```diff
@@ -0,0 +1,24 @@
+Deposit: event({
+    pubkey: bytes32,
+    withdrawal_shard_id: uint256,
+    withdrawal_addr: address,
+    randao_commitment: bytes32})
+
+
+used_pubkey: public(bool[bytes32])
+
+
+@public
+@payable
+def deposit(
+        pubkey: bytes32,
+        withdrawal_shard_id: uint256,
+        withdrawal_addr: address,
```

Comment:
I suggest renaming `withdrawal_addr` to  `withdrawal_address` to align with `ValidatorRecord`.

Reply:
Agree, will fix